examples/demo.py

```diff
@@ -42,35 +49,37 @@
 
 singleEvent = data_source.command(
     commands.GetEventById(
-        "case3:th2-scope:20221110122224000000000:def0e516-cc13-4fa6-8be6-2b0873297c9c"
+        "demo_book_1:th2-scope:20221226140719671764000:9c59694b-8526-11ed-8311-df33e1b504e4"
     )
 )
 multipleEvents = data_source.command(
     commands.GetEventsById(
         [
-            "case3:th2-scope:20221110122224000000000:def0e516-cc13-4fa6-8be6-2b0873297c9c",
-            "case3:th2-scope:20221110122324000000000:c3c5f7d5-e06d-4e23-98c9-7f36220d26db",
-        ]
-    )
-)
-
-single_message = data_source.command(
-    commands.GetMessageById("case3:arfq02fix30:2:20221111165012889502000:1668182272676097251")
-)
-multiple_messages = data_source.command(
-    commands.GetMessagesById(
-        [
-            "case3:arfq02fix30:2:20221111165012889502000:1668182272676097251",
-            "case3:arfq02fix30:2:20221111165252889876000:1668182272676097315",
+            "demo_book_1:th2-scope:20221226140719671764000:9c59694b-8526-11ed-8311-df33e1b504e4",
+            "demo_book_1:th2-scope:20221226140723967243000:9ee8edcc-8526-11ed-8311-df33e1b504e4",
+            "demo_book_1:th2-scope:20221226140724065522000:9ef7e1ed-8526-11ed-8311-df33e1b504e4",
         ]
     )
 )
 
-# We can get events without knowing their ids beforehands, using SSE requests from the server with GetEvents command:
+# # # Because namespace was taken down temporary lw-provider can't return messages.
```

Comment:
We provide this script as a demo example.
People, who see it, can don't have access to this namespace at all.

We just should show what functionality this library has and how to use it.
This demo script is our documentation!

Some code lines are able to don't work, but it doesn't matter. 
This file is example how to use our library and this file should be perfect.

Reply:
I understand. I thought users would have access to namespace.
Comments removed.